software/src/app.py

```diff
@@ -0,0 +1,23 @@
+from flask import Flask, render_template
+import json
+
+from random import randint
+
+app = Flask(__name__)
+
+DEBUG = True
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+@app.route('/data')
+def data():
+    data = {
+        'foo': randint(0, 9)
```

Comment:
This route is just acting as a GET api to grab our data. The actual web app will be a one pager, so no page reloads/routing needed.